objcToxTests/OCTSubmanagerFriendsTests.m

```diff
@@ -331,6 +331,11 @@ - (void)testFriendConnectionStatusChanged
     [self.submanager tox:self.tox friendConnectionStatusChanged:OCTToxConnectionStatusTCP friendNumber:kFriendNumber];
     XCTAssertEqual(friend.connectionStatus, OCTToxConnectionStatusTCP);
     XCTAssertTrue(friend.isConnected);
+
+    id mockNotificationCenter = OCMClassMock([NSNotificationCenter class]);
+    OCMStub([self.dataSource managerGetNotificationCenter]).andReturn(mockNotificationCenter);
+    [self.submanager tox:self.tox friendConnectionStatusChanged:OCTToxConnectionStatusUDP friendNumber:kFriendNumber];
+    OCMVerify([mockNotificationCenter postNotificationName:kOCTFriendConnectionStatusChangeNotificationName object:friend]);
```

Comment:
It seems that `friend` is for some reason captured by OCMock. Because friend has strong reference to Realm, it's get captured too. This leads to exception in next test. I'm not sure what to do with that, need more investigation.